packages/client/src/stores/sidePanel.js

```diff
@@ -32,11 +33,19 @@ export const createSidePanelStore = () => {
     }, 50)
   }
 
+  const setSidepanelState = bool => {
+    clearTimeout(timeout)
```

Comment:
We don't need to clear the timeout here - that's specifically for improving the opening/closing behaviour.